.env.example

```
@@ -0,0 +1,7 @@
+export DB_USERNAME=postgres
+export DB_PASSWORD=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USERNAME key

(UnorderedKey)